lib/l10n/app_en.arb

```diff
@@ -488,6 +488,8 @@
   "timeFrenzy": "Time frenzy",
   "timeFrenzyMistakes": "Track Time Frenzy mistakes",
   "timeFrenzyMistakesDesc": "Enable to save mistakes made in Time Frenzy",
+  "randomizeTzumegoOrientation" : "Randomize tzumego orientation",
```

Comment:
please, let's be consistent with naming. It says "tzumego" here, then "tsumego" below, and "task" elsewhere. Let's use "task" everywhere instead, including the setting key.